server/jetstream_cluster.go

```diff
@@ -3730,7 +3735,11 @@ func (js *jetStream) monitorConsumer(o *consumer, ca *consumerAssignment) {
 
 			// Check for migrations (peer count and replica count differ) here.
 			// We set the state on the stream assignment update below.
-			if isLeader && len(rg.Peers) != o.replica() {
+			replicas, err := o.replica()
+			if err != nil {
+				return
```

Comment:
For this one and others, not sure if it is ok to "return", or I should instead "continue" in that we would then consume the "quit" signal. Assuming that this is really what is happening... (that is, o.replica() was panic'ing because consumer was stopped and monitorConsumer() would have returned at the next select...

Reply:
based on the uncertainty of when this can happen, maybe do continue instead?

Reply:
Done